src/main/java/io/github/moulberry/notenoughupdates/overlays/FarmingSkillOverlay.java

```diff
@@ -88,6 +90,9 @@ public class FarmingSkillOverlay extends TextOverlay {
 
 	private String skillType = "Farming";
 
+	private static final Pattern CONTEST_AMOUNT_PATTERN = Pattern.compile(
+		" (Collected|(BRONZE|SILVER|GOLD|PLATINUM|DIAMOND) with) (?<amount>.*)");
```

Comment:
Can we be more explicit about the amount group and what it can contain? We have a try catch, but creating stacktraces often can lead to performance degradation